src/Uno.Toolkit.UI/Uno.Toolkit.UI.csproj

```diff
@@ -43,16 +43,21 @@
 	</ItemGroup>
 
 	<ItemGroup>
+	  <None Remove="Controls\ExtendedSplashScreen\ExtendedSplashScreen.xaml" />
 	  <None Remove="Controls\LoadingView\LoadingView.xaml" />
 	  <None Remove="Controls\SafeArea\SafeArea.xaml" />
 	</ItemGroup>
 
 	<ItemGroup>
+	  <UpToDateCheckInput Remove="Controls\ExtendedSplashScreen\ExtendedSplashScreen.xaml" />
 	  <UpToDateCheckInput Remove="Controls\LoadingView\LoadingView.xaml" />
 	  <UpToDateCheckInput Remove="Controls\SafeArea\SafeArea.xaml" />
 	</ItemGroup>
 
 	<ItemGroup>
+	  <Page Update="Controls\ExtendedSplashScreen\ExtendedSplashScreen.xaml">
+	    <SubType>Designer</SubType>
+	  </Page>
```

Comment:
I feel like these None, UpToDateCheckInput, and Page items don't need to be here

Reply:
I've removed these lines